hardhat.config.js

```diff
@@ -65,7 +65,7 @@ module.exports = {
       optimizer: {
         enabled: true,
         // https://docs.soliditylang.org/en/v0.8.10/internals/optimizer.html#:~:text=Optimizer%20Parameter%20Runs,-The%20number%20of&text=A%20%E2%80%9Cruns%E2%80%9D%20parameter%20of%20%E2%80%9C,is%202**32%2D1%20.
-        runs: 1000000,
+        runs: 1000000
```

Comment:
this thing doesn't make any sense

Reply:
I think the move is to deploy at 10000, and in the readme note that tests need to be run at 1000000.

Meanwhile I've opened an issue with hardhat here https://github.com/NomicFoundation/hardhat/issues/2657